website/docs/intro/install/freebsd.mdx

```diff
@@ -0,0 +1,37 @@
+---
+sidebar_position: 1
+sidebar_label: FreeBSD
+description: |-
+  Install OpenTofu on FreeBSD.
+---
+
+import CodeBlock from "@theme/CodeBlock";
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import FreeBSDConvenienceScript from "!!raw-loader!./freebsd-convenience.sh";
+
+# Installing OpenTofu on FreeBSD
+
+OpenTofu is available for [FreeBSD](https://pkgs.org/download/opentofu) and can be installed using the pkg system or by downloading the package directly.
+
+## Installing using the installer
```

Comment:
This is duplicating what's already present in the documentation, I don't know if it's necessary here. If the install script works with FreeBSD, pointing people to the appropriate page should be enough.

Reply:
install script need few changes for freebsd, It doesn't have support for port/pkg. 

Should i remove the `Installing using github` ?

Reply:
All distro has it's own script in current setup

Reply:
Perfect, thank you!